core/lib/libtomcrypt/sub.mk

```diff
@@ -145,11 +141,277 @@ srcs-$(_CFG_CORE_LTC_SM2_DSA) += sm2-dsa.c
+srcs-$(_CFG_CORE_LTC_RSA) += src/pk/pkcs1/pkcs_1_mgf1.c
+srcs-$(_CFG_CORE_LTC_RSA) += src/pk/pkcs1/pkcs_1_oaep_decode.c
+srcs-$(_CFG_CORE_LTC_RSA) += src/pk/pkcs1/pkcs_1_oaep_encode.c
+srcs-$(_CFG_CORE_LTC_RSA) += src/pk/pkcs1/pkcs_1_os2ip.c
+srcs-$(_CFG_CORE_LTC_RSA) += src/pk/pkcs1/pkcs_1_pss_decode.c
+srcs-$(_CFG_CORE_LTC_RSA) += src/pk/pkcs1/pkcs_1_pss_encode.c
+srcs-$(_CFG_CORE_LTC_RSA) += src/pk/pkcs1/pkcs_1_v1_5_decode.c
+srcs-$(_CFG_CORE_LTC_RSA) += src/pk/pkcs1/pkcs_1_v1_5_encode.c
+srcs-$(_CFG_CORE_LTC_RSA) += src/pk/rsa/rsa_decrypt_key.c
+srcs-$(_CFG_CORE_LTC_RSA) += src/pk/rsa/rsa_encrypt_key.c
+srcs-$(_CFG_CORE_LTC_RSA) += src/pk/rsa/rsa_export.c
+srcs-$(_CFG_CORE_LTC_RSA) += src/pk/rsa/rsa_exptmod.c
+srcs-$(_CFG_CORE_LTC_RSA) += src/pk/rsa/rsa_import.c
+srcs-$(_CFG_CORE_LTC_RSA) += src/pk/rsa/rsa_key.c
+srcs-$(_CFG_CORE_LTC_RSA) += src/pk/rsa/rsa_make_key.c
+srcs-$(_CFG_CORE_LTC_RSA) += src/pk/rsa/rsa_sign_hash.c
+srcs-$(_CFG_CORE_LTC_RSA) += src/pk/rsa/rsa_verify_hash.c
+
+srcs-$(_CFG_CORE_LTC_DH) += src/pk/dh/dh.c
+srcs-$(_CFG_CORE_LTC_DH) += src/pk/dh/dh_check_pubkey.c
+srcs-$(_CFG_CORE_LTC_DH) += src/pk/dh/dh_export.c
+srcs-$(_CFG_CORE_LTC_DH) += src/pk/dh/dh_export_key.c
+srcs-$(_CFG_CORE_LTC_DH) += src/pk/dh/dh_free.c
+srcs-$(_CFG_CORE_LTC_DH) += src/pk/dh/dh_generate_key.c
+srcs-$(_CFG_CORE_LTC_DH) += src/pk/dh/dh_import.c
+srcs-$(_CFG_CORE_LTC_DH) += src/pk/dh/dh_make_key.c
+srcs-$(_CFG_CORE_LTC_DH) += src/pk/dh/dh_set.c
+srcs-$(_CFG_CORE_LTC_DH) += src/pk/dh/dh_set_pg_dhparam.c
+srcs-$(_CFG_CORE_LTC_DH) += src/pk/dh/dh_shared_secret.c
+
+srcs-$(_CFG_CORE_LTC_ECC) += src/pk/ecc/ecc.c
+srcs-$(_CFG_CORE_LTC_ECC) += src/pk/ecc/ecc_find_curve.c
+srcs-$(_CFG_CORE_LTC_ECC) += src/pk/ecc/ecc_free.c
+srcs-$(_CFG_CORE_LTC_ECC) += src/pk/ecc/ecc_get_oid_str.c
+srcs-$(_CFG_CORE_LTC_ECC) += src/pk/ecc/ecc_make_key.c
+srcs-$(_CFG_CORE_LTC_ECC) += src/pk/ecc/ecc_set_curve.c
+srcs-$(_CFG_CORE_LTC_ECC) += src/pk/ecc/ecc_set_curve_internal.c
+srcs-$(_CFG_CORE_LTC_ECC) += src/pk/ecc/ecc_shared_secret.c
+srcs-$(_CFG_CORE_LTC_ECC) += src/pk/ecc/ecc_sign_hash.c
+srcs-$(_CFG_CORE_LTC_ECC) += src/pk/ecc/ecc_ssh_ecdsa_encode_name.c
+srcs-$(_CFG_CORE_LTC_ECC) += src/pk/ecc/ecc_verify_hash.c
+srcs-$(_CFG_CORE_LTC_ECC) += src/pk/ecc/ltc_ecc_is_point.c
+srcs-$(_CFG_CORE_LTC_ECC) += src/pk/ecc/ltc_ecc_is_point_at_infinity.c
+srcs-$(_CFG_CORE_LTC_ECC) += src/pk/ecc/ltc_ecc_map.c
+srcs-$(_CFG_CORE_LTC_ECC) += src/pk/ecc/ltc_ecc_mulmod.c
+srcs-$(_CFG_CORE_LTC_ECC) += src/pk/ecc/ltc_ecc_mulmod_timing.c
+srcs-$(_CFG_CORE_LTC_ECC) += src/pk/ecc/ltc_ecc_mul2add.c
+srcs-$(_CFG_CORE_LTC_ECC) += src/pk/ecc/ltc_ecc_points.c
+srcs-$(_CFG_CORE_LTC_ECC) += src/pk/ecc/ltc_ecc_projective_add_point.c
+srcs-$(_CFG_CORE_LTC_ECC) += src/pk/ecc/ltc_ecc_projective_dbl_point.c
+
+srcs-$(_CFG_CORE_LTC_ED25519) += src/pk/ec25519/ec25519_crypto_ctx.c
+srcs-$(_CFG_CORE_LTC_ED25519) += src/pk/ec25519/ec25519_export.c
+srcs-$(_CFG_CORE_LTC_ED25519) += src/pk/ec25519/tweetnacl.c
```

Comment:
These 3 were built when at least one of `_CFG_CORE_LTC_X25519` and `_CFG_CORE_LTC_ED25519` was enabled.

Reply:
You're right, I'll fixup the commit to use _CFG_CORE_LTC_EC25519 for this.